Resources/public/scss/modules/universal-discovery/_tab.nav.item.scss

```diff
@@ -1,19 +1,20 @@
 .c-tab-nav-item {
     border: 0 none;
-    border-radius: calculateRem(8px) calculateRem(8px) 0 0;
+    border-radius: calculateRem(4px) calculateRem(4px) 0 0;
     font-size: calculateRem(16px);
     line-height: calculateRem(30px);
     padding: calculateRem(4px) calculateRem(16px);
     outline: none;
     cursor: pointer;
     transition: background 0.3s $ez-admin-transition, color 0.3s $ez-admin-transition;
-    color: $ez-color-base-dark;
+    color: $ez-color-base-medium;
+    fill: $ez-color-base-medium;
```

Comment:
Unless I'm mistaken, the `fill` property works only when applied to SVG element itself.

Reply:
It may be inherited. 😏 😄 

I checked this in the browser and it works, at least in Chrome and Firefox. I also looked for specification and I found this for SVG: https://www.w3.org/TR/2011/REC-SVG11-20110816/painting.html#FillProperties
I also found out that we may use here `currentColor` value for fill: https://www.w3.org/TR/2018/REC-css-color-3-20180619/#currentcolor-def, also, as described in this article: https://css-tricks.com/cascading-svg-fill-color/ 

What do you think about starting using `currentColor` in out SCSS? It may simplify code and future changes. @sunpietro @dew326 ? 🙂

Reply:
I knew about the `currentColor` value. I have nothing against using it in our codebase :)